src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -141,7 +145,7 @@ public Test BuildFrom(IMethodInfo method, Test parentSuite)
                     tests.Add(test);
             }
 
-            return tests.Count > 0
+            return shouldParameterizeForFewerThanTwoTests || tests.Count > 0
```

Comment:
@CharliePoole we cannot do `if (tests.Count > 0 && builders.Count > 0)` due to this line: https://github.com/jnm2/nunit/blob/56283ed961bdb9f685864408208bd121c3132d2b/src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs#L140

For reasons I hope are obvious. 134 tests fail.